docs/emulator-suite/_includes/_rtdb_admin_quickstart.md

```diff
@@ -0,0 +1,6 @@
+The Firebase Admin SDKs automatically connect to the Realtime Database emulator
```

Comment:
Extracted into its own file. We can modify devsite to include this file as well. No actual changes.